packages/nextjs-mf/project.json

```diff
@@ -65,5 +65,5 @@
     }
   },
   "tags": [],
-  "implicitDependencies": ["node"]
+  "implicitDependencies": ["node", "utils"]
```

Comment:
Doesn't nx resolves the deps? Why do you need to maintain it?

Reply:
nx couldn't able to resolve it when we `serve or build` applications from `app` folder. Since the libraries are `webpack plugins`, nx can't figure out the deps.

`implicitDependencies` are added for local development.